src/main/java/com/osiris/autoplug/client/Server.java

```diff
@@ -397,7 +397,11 @@ public static void submitCommand(@NotNull String command) throws IOException {
                 os.write((command + System.lineSeparator()).getBytes(StandardCharsets.UTF_8));//TERMINAL.writer().write(command + System.lineSeparator());
             os.flush();
         } else {
-            AL.warn("Failed to submit command '" + command + "' because server is not running!");
```

Comment:
keep old code, to always show command, to ensure that in cases there might be whitespace or hidden chars this could be useful.